Gluntch, the Bestower.txt

```diff
@@ -0,0 +1,15 @@
+Name:Gluntch, the Bestower
+ManaCost:1 G W
+Types:Creature Jellyfish
+PT:0/5
+K:Flying
+SVar:X:PlayerCountPlayers$Amount
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigChoosePlayer1 | TriggerDescription$ At the beginning of your end step, choose a player. They put two +1/+1 counters on a creature they control. Choose a second player to draw a card. Then choose a third player to create two Treasure tokens.
+SVar:TrigChoosePlayer1:DB$ ChoosePlayer | Defined$ You | Choices$ Player | RememberChosen$ True | ChoiceTitle$ Choose a player to put two +1/+1 counters on a creature they control | SubAbility$Counters

```

Comment:
For this and all other SubAbility$ just put a space for style/search

eg `SubAbility$Counters` -> `SubAbility$ Counters`